tasks/create-product-sbom/create-product-sbom.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: create-product-sbom
+  labels:
+    app.kubernetes.io/version: "1.0.0"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: release
+spec:
+  description: >-
+    Create product-level SBOM from release notes.
+  params:
+    - name: dataJsonPath
+      description: Path to the JSON data.
+      type: string
+  workspaces:
+    - name: data
+      description: Workspace to save the product-level SBOM to.
+  results:
+    - name: productSBOM
+      description: Path to the created product-level SBOM in the workspace.
+  steps:
+    - name: create-sbom
+      # TODO: replace tag with one containing the script
+      image: quay.io/konflux-ci/release-service-utils:35bffc8f17a841a76a247b9ee85ce727f31f7238
+      script: |
+        #!/usr/bin/env sh
+        create-product-sbom --data-path $(params.dataJsonPath) \
```

Comment:
You call this without the `.py` suffix, but currently this PR does not add that as a symlink: https://github.com/konflux-ci/release-service-utils/pull/254

(It probably should, for consistency with other scripts.)

Reply:
resolved